src/ansi.rs

```diff
@@ -563,6 +563,10 @@ pub enum NamedColor {
     BrightForeground,
     /// Dim foreground
     DimForeground,
+    /// Color for the selected text
+    SelectionText,
+    /// Color for the selection itself
+    Selection,
```

Comment:
To be in line with all other variables, this should be `SelectionBackground`, which is a little more explicit too.